docker/prod/push_image.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Usage: examples
+#    ./push_image.sh
+#    BRANCH_HEAD_TAG='git describe' ./push_image.sh
+#    REPO_OWNER=stanislavchlebec BRANCH_HEAD_TAG=`git describe` ./push_image.sh
+#    LOCAL_IMAGE='prod_vpp_agent:latest' DEFAULT_IMAGE_NAME='vpp-agent' ./push_image.sh
+
+# Warning: use only IMMEDIATELY after docker/dev/build.sh to prevent INCONSISTENCIES such as 
+#          a) after building image you switch to other branch which will result in mismatch of version of image and its tag
+#          b) you do not build the new image but only simply run this script which will result in mismatch version of image and its tag because the image is older than repository 
+LOCAL_IMAGE='prod_vpp_agent:latest' DEFAULT_IMAGE_NAME='vpp-agent' ../dev/push_image.sh
```

Comment:
Forgot to change here. Perhaps the push script should be only one and located in `docker` directory directly.

Reply:
I would prefer to leave it like it is.
The user then does not have to write variables before script
so IMO it has sense to make it like it is.

Reply:
The next thing is that how it is described in the beginning of the script there is close connection to the build.sh script - they IMHO should be one next to other one.

Reply:
Okay then. `DEFAULT_IMAGE_NAME` needs to be fixed anyway though.

Reply:
Yes - that is what you refered that I forgot:-) I did not realised.
I do it.